uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/BAO/AbsenceTypeTest.php

```diff
@@ -681,4 +687,80 @@ public function testCalculateToilExpiryDateWhenAbsenceTypeAllowsAccrualsRequestA
     $expiry = $absenceType2->calculateToilExpiryDate($date);
     $this->assertEquals('2017-09-10', $expiry->format('Y-m-d'));
   }
+
+  public function testNonExpiredToilRequestsAreDeletedWhenToilIsDisabledForAbsenceType() {
```

Comment:
I think this method name is a bit confusing. It says it tests that "Non expired TOIL Request are deleted", but the assertions are all about asserting that expired things were not deleted. Maybe you should change the test name or the assertions?